models/cam-fv/shell_scripts/cesm2_1/DART_config.template

```diff
@@ -71,7 +71,8 @@ echo "==================="
 
 setenv DARTROOT               your_dart_path
 setenv DART_SCRIPTS_DIR       $DARTROOT/models/cam-fv/shell_scripts/your_setup_script_dir
-echo "DART_SCRIPTS_DIR = $DART_SCRIPTS_DIR"
+setenv DART_SCRIPT            your_dart_script
```

Comment:
For the 'your_dart_script' variable.  Is this ever going to be something other than 'assimilate.csh'  or 'no_assimilate.csh'?  I'm missing what the added flexibility here is for, would someone have variations on assimilate.csh?


sed -e "s#your_dart_script#assimilate.csh#"

Reply:
I generalized this because perfect_model.csh is now one of the possibilities.
I can imagine people using variations of assimilate.csh, but that wasn't my goal.

Reply:
oh ok, so  the flexibility of 'your_dart_script' is for assimilate.csh or perfect_model.csh